core/base/inc/TDirectory.h

```diff
@@ -303,7 +303,7 @@ can be replaced with the simpler and exception safe:
    virtual void        WriteKeys() {}
 
    static Bool_t       Cd(const char *path);
-   static void         DecodeNameCycle(const char *namecycle, char *name, Short_t &cycle, const size_t namesize = 0);
+   static size_t DecodeNameCycle(const char *namecycle, char *name, Short_t &cycle, const size_t namesize = 0);
```

Comment:
```suggestion
   static size_t       DecodeNameCycle(const char *namecycle, char *name, Short_t &cycle, const size_t namesize = 0);
```
please align `Decode` and `Cd`